chain/x/mint/simulation/genesis.go

```diff
@@ -86,9 +86,20 @@ func RandomizedGenState(simState *module.SimulationState) {
 
 	mintDenom := sdk.DefaultBondDenom
 	blocksPerYear := uint64(60 * 60 * 8766 / 5)
-	params := types.NewParams(mintDenom, inflationRateChange, inflationMax, inflationMin, goalBonded, blocksPerYear, mintAir)
+	mintMax := sdk.Coins{}
+
+	params := types.NewParams(
+		mintDenom,
+		inflationRateChange,
+		inflationMax,
+		inflationMin,
+		goalBonded,
+		mintMax,
+		blocksPerYear,
+		mintAir,
+	)
 
-	mintGenesis := types.NewGenesisState(types.InitialMinter(inflation), params)
+	mintGenesis := types.NewGenesisState(types.InitialMinter(inflation), params, types.MintPool{})
```

Comment:
No need to pass mint pool here, create a function to define a default empty mint pool (check oracle pool)